.circleci/config.yml

```diff
@@ -6,25 +6,26 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: dependency-cache-{{ checksum "packages/cf.js/package.json" }}{{ checksum "packages/contracts/package.json"}}{{ checksum "packages/iframe-wallet-poc/package.json"}}{{ checksum "packages/machine/package.json" }}{{  checksum "packages/playground/package.json" }}{{ checksum "packages/typescript-typings/package.json" }}
+          key: dependency-cache-{{ checksum "packages/cf.js/package.json" }}{{ checksum "packages/contracts/package.json"}}{{ checksum "packages/iframe-wallet-poc/package.json"}}{{ checksum "packages/machine/package.json" }}{{  checksum "packages/playground/package.json" }}{{ checksum "packages/typescript-typings/package.json" }}{{ checksum "packages/node/package.json" }}
       - run: yarn
       - save_cache:
-          key: dependency-cache-{{ checksum "packages/cf.js/package.json" }}{{ checksum "packages/contracts/package.json"}}{{ checksum "packages/iframe-wallet-poc/package.json"}}{{ checksum "packages/machine/package.json" }}{{  checksum "packages/playground/package.json" }}{{ checksum "packages/typescript-typings/package.json" }}
+          key: dependency-cache-{{ checksum "packages/cf.js/package.json" }}{{ checksum "packages/contracts/package.json"}}{{ checksum "packages/iframe-wallet-poc/package.json"}}{{ checksum "packages/machine/package.json" }}{{  checksum "packages/playground/package.json" }}{{ checksum "packages/typescript-typings/package.json" }}{{ checksum "packages/node/package.json" }}
```

Comment:
Does this thing support globs?

Reply:
Removed this formatting completely as CircleCI yaml parsing doesn't support interpreting globs.

https://github.com/counterfactual/monorepo/pull/235